### DISCUSSION TITLE
The developer never answered my submitted issues!

### DISCUSSION THREAD

**Discussion by ghost** [royqh1979/RedPanda-CPP#101] [General]
Am I non-existent to him? I don't think so. Other issues submitted by other people including issues in Chinese are not answered too. So I dismissed the idea that he ignored issues in English. Is this repo kind of a mirror and real development done on somewhere else? I think it's very likely!

**Comment by ccsugar** [comment]
sadly...but the developer has a QQ group,you can have a try,it is 367538953
